pynamodb/attributes.py

```diff
@@ -376,65 +348,42 @@ class BinarySetAttribute(SetMixin, Attribute[Set[bytes]]):
 
     def serialize(self, value):
         """
-        Returns a base64 encoded binary string
+        Returns a list of base64 encoded binary strings. Encodes empty sets as "None".
         """
-        if value and len(value):
-            return [b64encode(val).decode(DEFAULT_ENCODING) for val in sorted(value)]
-        else:
-            return None
+        return [b64encode(v).decode(DEFAULT_ENCODING) for v in value] if value else None
```

Comment:
removes the sorting and merges the `if value and len(value)` check

Reply:
```suggestion
        return [b64encode(v).decode(DEFAULT_ENCODING) for v in value] or None
```

Reply:
that throws a type error when value is None: `TypeError: 'NoneType' object is not iterable`